js/base/Exchange.js

```diff
@@ -1715,7 +1727,13 @@ module.exports = class Exchange {
                 }
             }
         }
-        return networkId;
+        if (this.options['hasUniqueNetworkIds']) {
```

Comment:
@ttodua this option does not seem very "self-explanatory", what if instead of the option we checked the size of `networkChainIdsByNames/networkNamesByChainIds` ? If bigger than 0, means we have this logic, wdyt?